KomaMRIFiles/Project.toml

```diff
@@ -9,14 +9,14 @@ HDF5 = "f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"
 KomaMRIBase = "d0bc0b20-b151-4d03-b2a4-6ca51751cb9c"
 MAT = "23992714-dd62-5051-b70f-ba57cb901cac"
 MRIFiles = "5a6f062f-bf45-497d-b654-ad17aae2a530"
+NIfTI = "a3a9e032-41b5-5fc4-967a-a6b7a19844d3"
```

Comment:
Remove.

Reply:
Resolved in [7f1cc52](https://github.com/JuliaHealth/KomaMRI.jl/pull/184/commits/7f1cc52925d3be8803c4ff0c95c1e06f391b7c83)